src/CodeGen_C.cpp

```diff
@@ -531,6 +531,29 @@ class ExternCallPrototypes : public IRGraphVisitor {
 };
 }
 
+void CodeGen_C::forward_declare_type(const Type &t) {
+    auto handle_type = t.handle_type;
+    if (!handle_type) return;
```

Comment:
When the rash of SSL errors hit a while back, I decided to never write if statements without braces. I also think it is clearer if the two returns are done in one statement.